recipes/aptardi/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
I would suggest deleting this file altogether, in favor of the `build: script:` directive in `meta.yaml`.
```suggestion
```